src/app/pages/overlays/complex-bottom-sheet/complex-bottom-sheet.component.html

```diff
@@ -0,0 +1,41 @@
+<mat-toolbar color="primary">
+    <button *ngIf="isSmall" style="margin-left: -8px;" mat-icon-button (click)="toggleMenu()">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title">Complex Bottom Sheet</div>
+    <pxb-spacer></pxb-spacer>
+    <button mat-icon-button (click)="showBottomSheet()">
+        <mat-icon>more_vert</mat-icon>
+    </button>
+</mat-toolbar>
+<mat-list *ngIf="data && data.length > 0">
+    <pxb-info-list-item
```

Comment:
Looks like something goofy is happening with the ListItems again (might be box-sizing issue again):
Should look like:
![image](https://user-images.githubusercontent.com/29152776/87440317-d4edc880-c5bf-11ea-928a-640fef55353d.png)

Note the icons are vertically centered with the text and left aligned to the edge of the avatar

Currently looks like:

![image](https://user-images.githubusercontent.com/29152776/87440410-f058d380-c5bf-11ea-9d6e-47acf5ed560c.png)

Reply:
From a design point of view I personally like the second one better (did I do that in the React mono repo too?)

Reply:
For some reason, there was padding on the icons...

Reply:
@huayunh The react mono repo example has the icons aligned like the second picture

Reply:
The first picture is how the ILI _should_ look (that is, Eric said it). Everything is aligned on the Material keylines. Ian and I have gone back and forth on this a bunch in the past. Center aligning makes it so the icons don't align with anything else in the application. I agree it tends to look nicer in this one case where you have a mix of icons and avatars, but I would advocate for sticking with the left alignment in the majority of cases (and the default). You can achieve the center aligning (as we have done in the React example) by using a transparent avatar background.